File: src/python/TractorGame.py
from Player import Player
from Round import Round

class TractorGame:

  NUM_PLAYERS = 4
  NUM_DECKS = 2
  
  def __init__(self):
    self.startGame()
    
  def startGame(self):
    self.currentRound = None
    Player.players = []

    for i in range(TractorGame.NUM_PLAYERS): 
      Player.players.append(Player(i))
  
  def nextRound(self):
      if self.currentRound == None: 
        self.currentRound = Round(Player.players[0])
      else: 
        self.currentRound = Round(Player.players[self.currentRound.getWinner()])
      return self.currentRound

File: src/tractor/TractorGame.java
package tractor;

public class TractorGame {

	static final int NUM_PLAYERS = 4;
	static final int NUM_DECKS = 2;
	public static final boolean TESTING = true;
	static Player[] players;

	public static void startGame() {

		Round currentRound = null;
		players = new Player[NUM_PLAYERS];

		for (int i = 0; i < players.length; i++) {
			players[i] = new Player(i);
		}

		while (true) {
			if (currentRound == null) {
				currentRound = new Round(players[0]);
			} else {
				currentRound = new Round(players[currentRound.getWinner()]);
			}
			currentRound.start();
		}
	}

	public static void main(String[] args) {
		startGame();
	}
}
